### DISCUSSION TITLE
Can we use Windi CSS instead of Tailwind?

### DISCUSSION THREAD

**Discussion by JohanHeyvaert** [themesberg/flowbite#386] [Ideas]
**Is your feature request related to a problem? Please describe.**
I've just learned about [Windi CSS](https://windicss.org) and it looks like a superior alternative to Tailwind. Most importantly (to me anyway): it offers better performance (for developers and users). 
(BTW, an interesting read: https://antfu.me/posts/reimagine-atomic-css)

Do you think that Flowbite can be used together with Windi (instead of Tailwind)? I don't know if/which modifications that would require. Maybe it just works out of the box, but if so I don't know if it would have hidden side effects.

**Describe the solution you'd like**
Allow Flowbite to work with Windi CSS. 🥳 

**Describe alternatives you've considered**
Keep on using Tailwind. 😉

**Comment by zoltanszogyenyi** [comment]
Hello @JohanHeyvaert,

Thanks for opening up the discussion!

Flowbite is and most likely will keep being a superset of Tailwind CSS - but one of our community members created a [Windi CSS plugin for Flowbite](https://github.com/AlessioGr/flowbite-windicss-plugin).

Perhaps you can check that out!

I'll also move this to the discussions area instead of keeping it an issue.

Cheers,
Zoltan
